docs/faq.md

```diff
@@ -153,19 +189,23 @@ for al_step in range(NUM_AL_STEP):
     model.test_on_dataset(...)
     # Label the next set of labels.
     loop.step()
-    
+
 ```
 
 #### Use Bayesian model average when testing.
 
-When using MC-Dropout, or any other Bayesian methods, you will want to compute the Bayesian model average (BMA) at test time too.
+When using MC-Dropout, or any other Bayesian methods, you will want to compute the Bayesian model average (BMA) at test
+time too.
 
-To do so, you can specify the `average_predictions` parameters in `ModelWrapper.test_on_dataset`. The prediction will be averaged over `iterations` stochastic predictions. 
+To do so, you can specify the `average_predictions` parameters in `ModelWrapper.test_on_dataset`. The prediction will be
+averaged over `iterations` stochastic predictions.
 
 This will slightly increase the ECE of your model and will improve the predictive performance as well.
 
```

Comment:
maybe we can add a note on recently available `unpatch` module?

Reply:
Done above

Reply:
Added above